seed/static/seed/partials/delete_cycle_modal.html

```diff
@@ -49,6 +53,6 @@ <h4 class="modal-title" translate>Delete Cycle?</h4>
     </div>
 </div>
 <div class="modal-footer">
-    <button type="button" class="btn btn-default" ng-click="cancel()">{$delete_cycle_success == null ? 'Cancel' : 'Close'$}</button>
-    <button ng-if="cycle_has_inventory == false && delete_cycle_success == null" type="button" class="btn btn-danger" ng-click="confirmDelete()" autofocus translate>Yes, Delete Cycle</button>
+    <button type="button" class="btn btn-default" ng-click="cancel()">{$ delete_cycle_status == 'success' ? 'Close' : 'Cancel' $}</button>
```

Comment:
I believe that hitting `Cancel` wouldn't actually Cancel the deletion. If that's right, then I'd vote to not even show the `Cancel` button once this has kicked off. If we do that, it might be worth to disabling the click-away or keyboard escape like we do for owner org deletion here: https://github.com/SEED-platform/seed/pull/2620/files#diff-2705f5db618213153e520ff3d70128459ca769b8c0034414a2b4522235fdf3b1R130-R131

Reply:
ah nice, I like it